Sources/VRMSceneKit/Extensions/SCNQuaternion+.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  SCNQuaternion+.swift
+//  VRMSceneKit
+//
+//  Created by Tomoya Hirano on 2019/12/28.
+//  Copyright © 2019 tattn. All rights reserved.
+//
+
+import SceneKit
+
+/*
+ x, y, and z represent the imaginary values.
+ */
+@inlinable func SCNQuaternionMake(_ x: SCNFloat, _ y: SCNFloat, _ z: SCNFloat, _ w: SCNFloat) -> SCNQuaternion {
```

Comment:
I think `***Make()`s are legacy functions for Objective-C, like `CGRectMake`.
Why do you add the function?

Reply:
Because I used GLKit as reference.
I'll fix legacy style codes for more swifty.

Reply:
I see, thanks! 😄